### PR TITLE
Version 1.0.0

### PR DESCRIPTION
**Pull request by Scotchester**
Establishes a new canonical location for CFPB's color, font, and CF styling overrides.

**Review, please:**
- @contolini
- @jimmynotjim
- @ascott1
- @cfpb/front-end-team-admin

### CODE REVIEW

**Comment by awolfe76** [comment]
:+1: Looks good to me. Will be really useful right away!

File: package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "cf-theme-cfpb",
+  "version": "1.0.0",
+  "description": "Capital Framework theme for CFPB websites and applications.",
+  "less": "src/theme.less",
```
Comment (contolini) [inline package.json]: This should be updated to `cf-theme-cfpb.less`.

File: src/color-palette.less [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+ * Run the list of project-wide Find and Replace actions given below
+ * to update deprecated variables to their current equivalents.
+ *
+ * IMPORTANT: You must execute these in the specified order
+ * to ensure that you don't overwrite previous updates.
+ *
+ * ALSO IMPORTANT: When running replacements #3, #7, and #19, ensure that you
+ * have the regular expression option turned on in your editor.
+ * Failure to do so will replace the partial match in every tint of that color.
+ */
```
Comment (contolini) [inline src/color-palette.less:111]: This four-paragraph disclaimer is really intimidating but I'm guessing it's rare someone will have to update the colors.

File: package.json (hunk 1 above)
Comment (Scotchester) [inline package.json]: Good catch!

File: src/color-palette.less (hunk 2 above)
Comment (Scotchester) [inline src/color-palette.less:111]: Yeah, and it's important to get it right.

**Comment by Scotchester** [comment]
Fixed the one issue. It's go time! :shipit:

**Pull request merged**